Platform/Sophgo/SG2042Pkg/Universal/Dxe/RamFvbServicesRuntimeDxe/FvbServicesRuntimeDxe.inf

```diff
@@ -0,0 +1,84 @@
+## @file
+#  Component description file for RAM Flash Fimware Volume Block DXE driver

```

Comment:
Thank you for your feedback! The FvbServicesRuntimeDxe is required by VariableRuntimeDxe. And some modifications have been made to FwBlockService. c in the original module. The MarkMemoryRangeForRuntimeAccess function has been removed because it conflicts with memory initialization in the Sec module. We will try to see if there is a better solution.

Reply:
Thank you for your feedback! We will no longer be adding variable memory to the SEC stage.

Reply:
That's great!. Thank you.